keyboards/redox/proton_c/halconf.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define HAL_USE_SERIAL TRUE
+#define STM32_SERIAL_USE_USART1 TRUE
```

Comment:
`halconf.h` is a wrong place for this option — it needs to be placed in `mcuconf.h` (again, you could see an example in #14490).

Reply:
done. I missed the file it was supposed to be in when I read [this doc](https://docs.qmk.fm/#/serial_driver)